src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralConcluderTest.kt

```diff
@@ -49,7 +49,7 @@ internal class ReferralConcluderTest {
   fun `concludes referral as cancelled when ending a referral with no sessions attempted`() {
     val timeAtStart = OffsetDateTime.now()
     val actionPlan = actionPlanFactory.create(numberOfSessions = 2)
-    val referralWithActionPlanAndNoAttemptedSessions = referralFactory.createSent(actionPlan = actionPlan)
+    val referralWithActionPlanAndNoAttemptedSessions = referralFactory.createSent(actionPlans = mutableListOf(actionPlan))
```

Comment:
_nit_ factory could support both singular and list use cases, if that reduces cognitive load on reading the test; what do you think?

Reply:
i kinda figured that if referrals have lists of action plans, it might make it _harder_ to reason about tests that set individual action plans